redux/actions/currenciesActions.ts

```diff
@@ -44,3 +45,32 @@ export const addCurrency = (valueSearched: string) => async (dispatch: Dispatch<
     return error
   }
 }
+
+export const updateCurrencyList = () => async (dispatch: Dispatch<IGetCurrenciesAction>, getState: () => IState) => {
+  const fields = 'metrics?fields=id,symbol,name,market_data/price_usd,market_data/percent_change_usd_last_24_hours'
+  const {userCurrencyList} = getState().currencies
+  let temporaryList: ICurrency[] = []
+
+  const storeCurrencyList = async () => {
+    try {
+      await AsyncStorage.setItem(AsyncKeys.userCurrencyList, JSON.stringify(temporaryList))
+    } catch (error) {
+      console.log(error)
+    }
+  }
+
+  if(userCurrencyList.length > 0) {
+    userCurrencyList.map( currency => {
+      axios.get(`${HOST}/v1/assets/${currency.symbol.toLowerCase()}/${fields}`)
+        .then( response => {
+          if(response.data.data) temporaryList = [ ...temporaryList, response.data.data ]
```

Comment:
[suggestion] It would be better to just use `push()` the new data into the array